<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeScreen</class>
 <widget class="QMainWindow" name="homeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>360</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="9">
     <widget class="QLabel" name="userCompletedEventsLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="ratingLabel">
      <property name="text">
       <string>Rating:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8" rowspan="4" colspan="2">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="firstDayOfWeek">
       <enum>Qt::Sunday</enum>
      </property>
      <property name="gridVisible">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QCalendarWidget::SingleSelection</enum>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::SingleLetterDayNames</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::ISOWeekNumbers</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" rowspan="5" colspan="8">
     <widget class="QListWidget" name="taskList"/>
    </item>
    <item row="6" column="6">
     <widget class="QPushButton" name="deleteEventButton">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLabel" name="completedTasksLabel">
      <property name="text">
       <string>Completed Tasks: </string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Events</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="8">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Todays Tasks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addTaskButton">
        <property name="text">
         <string>Add Task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewTaskButton">
        <property name="text">
         <string>View Task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteTaskButton">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="completeButton">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="9">
     <widget class="QLabel" name="userRatingLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QLabel" name="userCompletedTasksLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QLabel" name="completedEventsLabel">
      <property name="text">
       <string>Completed Events: </string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="8">
     <widget class="QListWidget" name="eventsList"/>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Progress Tracker</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="viewEventButton">
      <property name="text">
       <string>View Event</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="addEventButton">
      <property name="text">
       <string>Add Event</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNotes">
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionCompleted_Tasks"/>
   </widget>
   <widget class="QMenu" name="menuNotes_2">
    <property name="title">
     <string>Notes</string>
    </property>
    <addaction name="actionNotes"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionMy_Account"/>
   </widget>
   <addaction name="menuNotes"/>
   <addaction name="menuNotes_2"/>
   <addaction name="menuAccount"/>
  </widget>
  <action name="actionCompleted_Tasks">
   <property name="text">
    <string>Completed Tasks</string>
   </property>
  </action>
  <action name="actionMy_Account">
   <property name="text">
    <string>My Account</string>
   </property>
  </action>
  <action name="actionNotes">
   <property name="text">
    <string>Notes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
